docs/user-guide/sdks-using.md

```diff
@@ -0,0 +1,113 @@
+# Using Zowe SDKs
+
+Leverage the Zowe Client Software Development Kits (SDKs) to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Node.js Client SDK
+- Zowe Python Client SDK
+
+## API documentation
+
+For details about the available API endpoints, browse or download the [API Reference documentation](https://docs.zowe.org/stable/#zowe-client-sdk-reference-guides).
+
+## Software requirements
+
+### Node.js
+
+If you install Node SDK packages from the online registry, the required dependencies are installed automatically.
+
+If you download Node SDK packages from Zowe.org, the folder contains dependencies that you must install manually. Extract the TGZ files from the folder, copy the files to your project, and issue the following command for each dependency:
```

Comment:
Would it make more sense for the last to read `issue the following commands to install the dependencies`? There are two commands listed, and they each install a dependency.

Reply:
Totally - fixing.